tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/commons/js/file-browser/file-browser.js

```diff
@@ -36,6 +36,43 @@ define(['jquery', 'backbone', 'lodash', 'log', /** void module - jquery plugin *
             this._showSamples = _.get(config, 'showSamples');
             this._showBundles = _.get(config, 'showBundles');
             this._multiSelect = _.get(config, 'multiSelect', false);
+            this._deleteIcon = _.get(config, 'deleteIcon', false);
+            var self = this;
+
+            /**
+             * custom plugin for jstree to add delete icon for each file in workspace-explorer.
+             */
+            (function ($, undefined) {
+                "use strict";
```

Comment:
What is the usage of this?

Reply:
80a9a2d this commit fix the above request change